sphinx-docs/usermanual/userman_admin.rst

```diff
@@ -204,30 +218,101 @@ Downloading Individual Videos
+
+        For example, on Windows:
+        CONTENT_ROOT="C:\\torrented_videos_location\\"
+
+        For example, on Linux:
+        CONTENT_ROOT="/home/me/torrented_videos_location/"
+
+#. Restart your server. If you are unsure on how to do this, please see `Restarting Your Server`_ . 
+
+.. _BitTorrent Sync: http://www.getsync.com/
+
 
 Language Packs
 ---------------------------
-KA Lite supports internationalization. You can downloa dlanguage packs for yoru language if it is available. A language pack comes with all the available subtitles and user interface translations for that language. When it is installed, KA Lite will giv eyou the option to download individual dubbed videos from the language's Khan Academy YouTube channel.
+KA Lite supports internationalization. You can download language packs for your language if it is available. A language pack comes with all the available subtitles and user interface translations for that language. When it is installed, KA Lite will give you the option to download individual dubbed videos from the language's Khan Academy YouTube channel.
 
 Download Language Packs
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 To download language packs: 
+
 #. From the "Manage" page, click on the "Language" tab. 
 #. Select the language pack you wish to download by selecting from the drop-down menu.
 #. Click the "Get Language Pack" button. 
-#. Once the download finishes, you can see your language pack in the list of installed packs. Students and coaches will now be able to switch their language to any of the installed language packs. Their default will be the default that you set by clicking on "Set as default". 
+#. Once the download finishes, you can see your language pack in the list of installed packs. Learners and coaches will now be able to switch their language to any of the installed language packs. Their default will be the default that you set by clicking on "Set as default". 
 
 Delete Language Packs
 ^^^^^^^^^^^^^^^^^^^^^^^^^
+To delete language packs:
+
 #. Log in as the administrator.
 #. Click the "Languages" link in the navigation bar
 #. In the Installed Languages section, there is a button for deletion of each language.
 
 
+Restarting Your Server
+-----------------------
+If you have made some configuration changes (such as changing the filepath to your video content to your liking), or if you feel the need to reboot your KA Lite system, you may want to restart your server. Please note that this will cause KA Lite to become inaccessible to any users. However, this will not delete any user accounts or information that you have configured during set up. 
+
+This process varies, depending on which OS you are running the KA Lite Server on. 
+
+Restarting Your Server: Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Linux
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Restarting Your Server: Mac 
```

Comment:
This is probably good to include, but the included `kalite` binary should work equally well on both linux and mac. Best to touch base with Ben Bach on the latest developments on the `kalite` command line interface, he's actively working on it.